ros/src/computing/planning/decision/packages/decision_maker/nodes/decision_maker/decision_maker_node_stateupdate.cpp

```diff
@@ -336,7 +336,20 @@ void DecisionMakerNode::updateStateStop(int status)
       layer_msg.data = "detectionarea";
       Pubs["filtering_gridmap_layer"].publish(layer_msg);
     }
-    if (current_velocity_ == 0.0 && !foundOtherVehicleForIntersectionStop_ && !timerflag)
+
+    // obtain distance from self to stopline -> TODO: refactor!
+    const static double STOPING_AREA_DISTANCE = 8.0;
```

Comment:
is this the distance stopline to base_link?

Reply:
yes.